src/appsettings.h

```diff
@@ -109,12 +122,19 @@ class AppSettings : public QObject
     QDateTime m_summaryUpdated;
     quint32 m_infoViewed;
     QString m_countryCode;
+    bool m_autoUpdate;
+    qint32 m_autoUpdateTime;
+    Notifications::NotifyLevel m_notifyLevel;
+    QList<quint32> m_notifyIds;
+
+    // Attenuation duration properties
     QList<double> m_riskWeights;
     qint32 m_defaultBuckeOffset;
     qint32 m_normalizationDivisor;
     QList<RiskScoreClass> m_riskScoreClasses;
-    bool m_autoUpdate;
-    qint32 m_autoUpdateTime;
 };
 
+QDataStream &operator<<(QDataStream &out, const QList<quint32> &list);
+QDataStream &operator>>(QDataStream &in, QList<quint32> &list);
```

Comment:
Couldn't it be `const QDataStream &in`?

Reply:
Intuitively that would make sense, but the input datastream is actually modified by the use of `>>` inside the method, and the overloaded `QDataSTream >>` operator has [no const version](https://git.sailfishos.org/mer-core/qtbase/blob/mer-5.6/src/corelib/io/qdatastream.h#L135). So it won't compile if this is set as `const` I'm afraid.

Reply:
Ok, then nevermind.